vendor.conf

```diff
@@ -20,7 +20,7 @@ github.com/opencontainers/runc 429a5387123625040bacfbb60d96b1cbd02293ab
 github.com/Microsoft/go-winio v0.4.1
 github.com/containerd/fifo 69b99525e472735860a5269b75af1970142b3062
 github.com/opencontainers/runtime-spec 198f23f827eea397d4331d7eb048d9d4c7ff7bee
-github.com/containerd/go-runc 5d38580d03f4fbf2f09b2b0065deeeaf8d21aac2
+github.com/containerd/go-runc df573a0e124617148f49720e1447061131675694 git://github.com/tonistiigi/go-runc.git
```

Comment:
nit: `https://` seems preferred over `git://` on github: https://stackoverflow.com/questions/19317264/does-github-support-git-protocol-for-pull